pkg/agent/types/annotations.go

```diff
@@ -24,6 +24,9 @@ const (
 	// NodeWireGuardPublicAnnotationKey represents the key of the Node's WireGuard public key in the Annotations of the Node.
 	NodeWireGuardPublicAnnotationKey string = "node.antrea.io/wireguard-public-key"
 
+	// NodeMaxEgressIPsAnnotationKey represents the key of the maximum number of Egress IPs in the Annotations of the Node.
```

Comment:
How about "the key of maximum Egress IP number"?

Reply:
updated